Corona-Warn-App/src/main/res/values-de/release_info_strings.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<resources xmlns:tools="http://schemas.android.com/tools" xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2" tools:ignore="MissingTranslation">
+    <!-- ####################################
+                 Release Info Screen 1.12
+     ###################################### -->
+
+    <!-- XHED: Title for the release info screen -->
+    <string name="release_info_header">"Neue Funktionen"</string>
+    <!-- XHED: Version title for the release info screen -->
+    <string name="release_info_version_title">"Release <xliff:g id="release_app_version" example="1.12">"%1$d"</xliff:g></string>
```

Comment:
You might want to copy this line to defaults - otherwise replacing the placeholder will crash on other languages

Reply:
Ok, I just realised it's in English anyway :)